src/modules/schnorrsig/main_impl.h

```diff
@@ -171,21 +154,26 @@ int secp256k1_schnorrsig_verify(const secp256k1_context* ctx, const secp256k1_sc
         return 0;
     }
 
+    if (!secp256k1_xonly_pubkey_load(ctx, &pk, pubkey)) {
+        return 0;
+    }
+
     secp256k1_schnorrsig_sha256_tagged(&sha);
     secp256k1_sha256_write(&sha, &sig->data[0], 32);
-    secp256k1_xonly_pubkey_serialize(ctx, buf, pk);
+    secp256k1_fe_normalize_var(&pk.x);
```

Comment:
I *think* this normalize isn't needed? (because it was just deserialized, and when serializing you *have* to normalize it to avoid malleabilities.)

Reply:
Indeed. Pubkeys are always normalized. Fixed.